test/gpu/native/studies/shoc/triadalt1.chpl

```diff
@@ -0,0 +1,210 @@
+                    B1= hos[elemsInBlock..#elemsInBlock];
+                }
+
+
+                var currentStream = true; // Representing Stream 1
+                                        // false represents stream 0
+                var currentIdx = 0;
+                // Loop corresponding to while loop in benchmark:
+                while(currentIdx < numMaxFloats) {
+                    // Copy back answer from last kernel call
+                    if(currentStream){
+                        // Stream 0 copy back
+                        hos[currentIdx..#elemsInBlock] = C0;
+                    } else {
+                        // Stream 1 copy back
+                        hos[currentIdx..#elemsInBlock] = C1;
+                    }
+
+                    currentIdx += elemsInBlock;
+
+                    if(currentIdx < numMaxFloats){
+                        kernelLaunches+=1;
+                        // We have more stuff to operate on
+                        if(currentStream){
+                            kernelTimer.start();
+                            // Launch on Stream 1
+                            forall i in 0..#elemsInBlock {
+                                C1[i] = A1[i] * alpha + B1[i];
+                            }
+                            kernelTimer.stop();
+                        } else {
+                            kernelTimer.start();
+                            // Launch on Stream 0
+                            forall i in 0..#elemsInBlock {
+                                C0[i] = A0[i] * alpha + B0[i];
+                            }
+                            kernelTimer.stop();
+                        }
+                    }
+
+                    if(currentIdx+elemsInBlock < numMaxFloats){
+                        // We have more stuff to copy over and then operate on
+                        // Copy back answer from last kernel call
+                        if(currentStream){
+                            // Stream 1 copy over
+                            A1 = hos[currentIdx+elemsInBlock..#elemsInBlock];
+                            B1 = hos[currentIdx+elemsInBlock..#elemsInBlock];
+                        } else {
+                            // Stream 0 copy over
+                            A0 = hos[currentIdx+elemsInBlock..#elemsInBlock];
+                            B0 = hos[currentIdx+elemsInBlock..#elemsInBlock];
+                        }
+                    }
+
+                    currentStream  = !currentStream;
+                }
+
+                timer.stop();
+                var timeElapsedInNanoseconds = timer.elapsed() * 1e9;
+                var triad = (numMaxFloats * 2.0) / timeElapsedInNanoseconds;
+                var bdwth = (numMaxFloats * numBytes(real(32)) * 3.0) // 4 is size of real(32)
+                /timeElapsedInNanoseconds;
+
+                if noisy {
+                    writeln("TriadFlops\t",
+                    // "Block:%{#####}KB\t".format(blkSize),
```

Comment:
I was running into the linker error which I don't anymore. I want to keep the non-formatted version of the output though for the noisy printing. This is so that in case anyone wants to look at the full precision for whatever reason they can check the noisy output because the result database output is formatted neatly. 

I will just delete the commented lines